tests/settings/test_settings_typo_tolerance_meilisearch.py

```diff
@@ -0,0 +1,62 @@
+DEFAULT_TYPO_TOLERANCE =   {
+    'enabled': True,
+    'minWordSizeForTypos': {
+        'oneTypo': 5,
+        'twoTypos': 9,
+    },
+    'disableOnWords': [],
+    'disableOnAttributes': [],
+}
+
+NEW_TYPO_TOLERANCE =   {
+    'enabled': True,
+    'minWordSizeForTypos': {
+        'oneTypo': 6,
+        'twoTypos': 10,
+    },
+    'disableOnWords': [],
+    'disableOnAttributes': ['title'],
+}
+
+def test_get_typo_tolerance_default(empty_index):
+    """Tests getting default typo_tolerance."""
+    response = empty_index().get_typo_tolerance()
+    assert isinstance(response, dict)
+    assert response == DEFAULT_TYPO_TOLERANCE
+    print(response)
+
+def test_update_typo_tolerance(empty_index):
+    """Tests updating typo_tolerance."""
+    index = empty_index()
+    response = index.update_typo_tolerance(NEW_TYPO_TOLERANCE)
+    assert isinstance(response, dict)
```

Comment:
Can you apply some testing practices like the ones we discuss here? https://github.com/meilisearch/integration-guides/issues/153

Reply:
Did you mean to just add some spaces between the different steps or also the naming? Also, I know I had to apply these testing practices but these new tests are in the same format that all the others. I had some trouble doing something different in just one place in a repository even if it's not really good practice.

Reply:
I see your point of view, and I totally agree with it in most cases, but in this particular case, you could do that without creating a mess with different styles.